service/grpc/codelingo/codelingo.pb.go

```diff
@@ -167,48 +111,6 @@ func (m *PathsFromOffsetRequest) String() string            { return proto.Compa
 func (*PathsFromOffsetRequest) ProtoMessage()               {}
```

Comment:
exported method PathsFromOffsetRequest.Descriptor should have comment or be unexported